taskfile/cache_test.go

```diff
@@ -0,0 +1,22 @@
+package taskfile_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/go-task/task/v3/taskfile"
+)
+
+func TestNewCache(t *testing.T) {
+	cacheDir := t.TempDir()
+	_, err := taskfile.NewCache(cacheDir)
+	require.NoErrorf(t, err, "creating new cache in temporary directory '%s'", cacheDir)
+}
+
+func TestNewCacheWithTTL(t *testing.T) {
```

Comment:
You don't have a need for this, I think.

You could update TestNewCache to accept a variadic and pass it to NewCache

Reply:
It could be a table-drive test, yeah. I'll boil it down a bit.

Reply:
0eadafa8d6fc309d694fef68c3b282180db86b95 should resolve.